code-scanning-export/src/main.ts

```diff
@@ -0,0 +1,65 @@
+import * as core from '@actions/core'
+import { getAlerts } from './api'
+import { createSarifReport } from './sarif'
+import * as fs from 'fs'
+import * as process from 'process'
+import * as path from 'path'
+
+type RepositoryInfo = {
+  owner: string
+  repo: string
+}
+
+/**
+ * The main function for the action.
+ * @returns {Promise<void>} Resolves when the action is complete.
+ */
+export async function run(): Promise<void> {
+  const repositoryInfo = getRepositoryInfo()
+  const ref = core.getInput('ref')
+
+  core.debug(
+    `Fetching open and dismissed alerts for repository ${repositoryInfo.owner}/${repositoryInfo.repo}#${ref}`
+  )
+
+  const alerts = await getAlerts(
+    repositoryInfo.owner,
+    repositoryInfo.repo,
+    ref,
+    core.getInput('token')
+  )
+
+  core.debug(`Found ${alerts.length} alerts, processing now...`)
+
+  const sarifReport = createSarifReport(alerts)
+  const filePath = path.join(process.cwd(), core.getInput('output-file'))
```

Comment:
```suggestion
  const filePath = path.isAbsolute(filePath) ? filePath : path.join(process.cwd(), core.getInput('output-file'))
```

The shared "setup" action sets up a directory in `/tmp` for release artifacts.  I'd like to be able to provide `$S3_ASSETS/sarif-report.json` (which expands to `/tmp/s3assets/sarif-report.json` to this action, and have the generated sarif report written to the path I've provided.